src/test/java/pl/project13/maven/git/UriUserInfoRemoverTest.java

```diff
@@ -0,0 +1,54 @@
+package pl.project13.maven.git;
+
+import static org.junit.Assert.assertEquals;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Created by ryan on 3/21/16.
+ */
```

Comment:
I try to not include any `@author` or `Created by` since it goes somewhat against collective ownership of code (if there's still any left in the code - let's remove those as well).

I'll clean that up.